### ISSUE TITLE
Links in PDF artifact do not work

### ISSUE THREAD

**Issue by jmgurney** [opencontainers/runtime-spec#1088]
The PDF downloaded from https://github.com/opencontainers/runtime-spec/releases/tag/v1.0.2 has links that do not work.

If I click on Introduction, the first bullet in the Table of Contents, MacOSX's preview pops up a dialog saying:
There is no application set to open the URL spec.md.

Please update your tools to either make intradocument links work, or just not create them.

Thanks.

**Comment by marshray** [comment]
This is true of the [released html version v1.3.0](https://github.com/opencontainers/runtime-spec/releases/tag/v1.3.0) as well, but worse perhaps as the resulting document contains far more links.

I had a look at the Pandoc documentation but I didn't see a quick and easy fix.

This [Stack Overflow question](https://stackoverflow.com/questions/40993488/convert-markdown-links-to-html-with-pandoc) attracted some potentially useful answers.

A trivial build solution seems elusive, as Pandoc is combining several `.md` files into a single doc. The `linux-features.md` document is referenced as `[linux-features.md](linux-features.md)`, but the anchor heading introducing this document is `<a name="linuxFeatures" />`.

An intra-doc link [`[...](config-linux.md#namespaces)`](https://github.com/opencontainers/runtime-spec/blob/92249139eea7161e13745abd4cb6d0ea02a3227a/features-linux.md?plain=1#L8) is used to refer to [`## <a name="configLinuxNamespaces" />Namespaces`](https://github.com/opencontainers/runtime-spec/blob/92249139eea7161e13745abd4cb6d0ea02a3227a/config-linux.md?plain=1#L20).
